Source/Spring2022_Capstone/Weapon/ShotgunWeapon.cpp

```diff
@@ -59,6 +59,10 @@ void AShotgunWeapon::Shoot()
 			}
 			
 			CurWeaponCharge += ShotCost;
+			if (OnWeaponChargeChangedDelegate.IsBound())
```

Comment:
There's no need to check if it's bound. Not sure if it's the same Execute() call but check on documentation to fix this.
Yes, I made the same mistake before and have kept an eye on it to fix it.